dm/pkg/conn/basedb.go

```diff
@@ -357,3 +370,37 @@ func (d *BaseDB) Close() error {
 
 	return err
 }
+
+// NeedsModernTerminology is for checking whether the database version requires modern terminolog.
+//
+// MySQL:
+//   - `SHOW MASTER STATUS` -> `SHOW BINARY LOG STATUS`
+//   - `SHOW SLAVE HOSTS`   -> `SHOW REPLICAS`
+//
+// MariaDB:
+//   - `SHOW MASTER STATUS` -> `SHOW BINLOG STATUS`
+//   - `SHOW SLAVE HOSTS`   -> `SHOW REPLICA HOSTS`
+func (d *BaseDB) NeedsModernTerminology() bool {
```

Comment:
also this one. (I didn't open IDE to check if other package are referring these symbols)